piptools/sync.py

```diff
@@ -181,8 +183,11 @@ def sync(
             for ireq in sorted(to_install, key=key_from_ireq):
                 click.echo("  {}".format(format_requirement(ireq)))
 
+        exit_code = 2
```

Comment:
```suggestion
        exit_code = 1
```

code `1` would be more appropriate to not clash with code `2`, which is used by click in the [usage error context](https://github.com/pallets/click/blob/123dd717439d8620d8d6be5574d2c9f007952326/click/exceptions.py#L51).